apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -90,19 +88,15 @@ func configureAPI(api *operations.PortLayerAPI) http.Handler {
 
 	api.TxtProducer = httpkit.TextProducer()
 
-	netCtx, err := network.NewContext(
-		net.IPNet{
-			IP:   net.IPv4(172, 16, 0, 0),
-			Mask: net.CIDRMask(12, 32),
-		},
-		net.CIDRMask(16, 32),
-		sess)
 	if err != nil {
-		log.Fatalf("failed to create network context: %s", err)
+		log.Fatalf("ExecHandler ERROR: %s", err)
```

Comment:
I think this is unreachable as err is not set by any of the calls between here and the check after session.NewSession